### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Create application pool_Keyboard:Integrated and classic controls are not accessible by keyboard under pipline section

### ISSUE THREAD

**Issue by Amlan454** [Microsoft/IIS.WebManager#84]
Environment Details:
URL: https://manage.iis.net/settings/servers
Browser: Edge(42.17134.1.0)Chrome(68.0.3440.106)
Keyboard.
MAS ID Violated: MAS 2.1.1

Repro Steps:
1.Open "Microsoft IIS" application using the URL: https://manage.iis.net/
2.Download and install "Microsoft IIS Administration" application.
3.Tab till Microsoft IIS  on the top navigation bar and press enter
4.Tab till Web server and press enter
4.Tab till Application pool and press enter
5.Tab till Create application pool Dropdown and press enter
6.Tab till Setting drop down menu and press enter
7.Check under pipeline section integrated and classic control is accessible by keyboard or not

Actual Result:
Integrated and classic controls are not accessible by keyboard under pipeline section

Expected Result:
Each control should be accessible by keyboard

User Impact:
Keyboard only users find difficulty in using the application if the controls are not accessible by keyboard

Mas Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}
![bug_84_integrated and classic controls are not accessible by keyboard](https://user-images.githubusercontent.com/42638683/44583404-cc6d2880-a7c2-11e8-822c-e4e3f19228a2.png)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #Keyboard; #MAS2.1.1; #C+E-H; #.NETWebsites; #Chrome; #Edge

**Comment by fsteffi** [comment]
#A11ySev2;

**Comment by Yashi1201** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
